src/preload.py

```diff
@@ -389,6 +377,38 @@ def list_images(image):
     return _list_images(image)
 
 
+def get_docker_init_file_content(image):
+    with mount_context_manager(image, 2) as mountpoint:
+        path = os.path.join(
+            mountpoint,
+            'lib',
+            'systemd',
+            'system',
+            'docker.service',
```

Comment:
This script is for preloading resinOS images, I don't think we care about anything else.
I'm not sure that checking for folders existence would be more reliable (at least for resinOS).
To handle other cases we could add a `--force-storage-driver <driver>` option.